### PR TITLE
Version 1.1

### LINKED ISSUE #8: Add string type state support
#### Description
Until now pyrandwalk just accept numbers(`int`s) as its states, it would be nice if we add other types support too.

For example we can take into consider a BiGram simulation like bellow:
#### Steps/Code to Reproduce
```
>>> lexicon = ['I', 'eat', 'food']
>>> bigram = np.array([[0.1, 0.8, 0.1], [0.3, 0.1, 0.6], [0.6, 0.3, 0.1]])
>>> rw = RandomWalk(lexicon, bigram)
>>> states, probs = rw.run()
```
#### Expected Behavior
```
>>> states
['I', 'eat', 'food', 'food', 'I', 'food', 'I', 'eat', 'food', 'I', 'eat']
>>> probs
[0.3, 0.8, 0.6, 0.1, 0.6, 0.1, 0.1, 0.8, 0.6, 0.6, 0.8]
```
#### Actual Behavior
```
Traceback (most recent call last):
    ...
IndexError: only integers, slices (`:`), ellipsis (`...`), numpy.newaxis (`None`) and integer or boolean arrays are valid indices
```
#### Operating System
Linux 5.4.0-80-generic #90-Ubuntu SMP Fri Jul 9 22:49:44 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux

#### Python Version
Python 3.8.10
#### pyrandwalk Version (Use : `pyrandwalk.__version__`)
1.0

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
close #8 
related to #9 and #2

### Added
- all input `state` type support
### Changes
- `make_prob_dist` modified
- new examples
- `paper.bib`
- `paper.md`

### CODE REVIEW

**Comment by sadrasabouri** [comment]
@sepandhaghighi 
If you have time, it would be nice if you leave a review for this version until 26th August.
But feel free it's not that much important.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [1.1] - 2021-08-26
+### Added
+- all input `state` type support
+### Changes
+- `make_prob_dist` modified
+- new examples
```
Comment (sepandhaghighi) [inline CHANGELOG.md:11]: I think it would be better if move this line and the 2 lines below to the `### Added` section

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (sadrasabouri) [reply CHANGELOG.md:11]: Yes you're right.
That issue occurred when merging.

File: Document/Document.ipynb [hunk 2]
```diff
@@ -116,9 +117,14 @@
...
     {
-     "output_type": "stream",
      "name": "stdout",
-     "text": "/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead\n  from cryptography.utils import int_from_bytes\n/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead\n  from cryptography.utils import int_from_bytes\n"
+     "output_type": "stream",
+     "text": [
+      "/usr/lib/python3/dist-packages/secretstorage/dhcrypto.py:15: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead\r\n",
+      "  from cryptography.utils import int_from_bytes\r\n",
+      "/usr/lib/python3/dist-packages/secretstorage/util.py:19: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead\r\n",
+      "  from cryptography.utils import int_from_bytes\r\n"
```
Comment (sepandhaghighi) [inline Document/Document.ipynb:126]: Please take a look at this warning in the next release!

**Review by sepandhaghighi — APPROVED** [review]
LGTM 💯

File: Document/Document.ipynb (hunk 2 above)
Comment (sadrasabouri) [reply Document/Document.ipynb:126]: That's ODD. which function is using this? :smile:

**Pull request merged**